noxfile.py

```diff
@@ -49,6 +50,8 @@ def lint(session):
     session.run("ruff", "check", "rustworkx", "retworkx", "setup.py")
     session.run("cargo", "fmt", "--all", "--", "--check", external=True)
     session.run("python", "tools/find_stray_release_notes.py")
+    session.run("typos", "--exclude", "releasenotes")
+    session.run("typos", "--no-check-filenames", "releasenotes")
```

Comment:
Ironically, releasenotes/ is one of the places I'd like to check for typos the most

Reply:
`releasenotes/` is checked, but not the filenames, because they the hashes from `reno` produce many false positives.

It may be possible to exclude file name checking for all YAML files or even YAML files within `releasenotes/` by adjusting `typos.toml`. The split command looked less magic to me.